src/sandbox/sandbox.h

```diff
@@ -0,0 +1 @@
+int lo_up();
```

Comment:
You should put the function comment here rather than in the .c file; this is the interface that callers would read.